ert_shared/status/tracker/evaluator.py

```diff
@@ -215,11 +204,11 @@ def request_termination(self):
             return
 
         with create_ee_monitor(
```

Comment:
Good approach! Can this create function take directly `ee_config` as parameter too?

Reply:
Yes, implemented it.
The same is likely true for `SyncWebsocketDuplexer`. Looking into this now.